tests/tests_pytorch/strategies/test_deepspeed_strategy.py

```diff
@@ -734,9 +733,7 @@ def test_deepspeed_multigpu_stage_3_checkpointing(tmpdir, automatic_optimization
     trainer.fit(model, datamodule=dm)
 
     results = trainer.test(datamodule=dm)
-    assert results[0]["test_acc"] > 0.7
```

Comment:
I've removed the convergence check.

It's really arbitrary and PyTorch doesn't even guarantee performance between versions

These kinds of tests should be benchmark tests. And ideally something like the same model trained with DeepSpeed vs normally

Reply:
I agree that they should be! Created #13684 to eventually put these kinds of tests back in benchmarks and also to gather other benchmarks buried in our tests if any exist.)

Reply:
There should be no interactions between standalone tests by design (reason why they are called standalone). 
It must be because you removed `seed_everything(1)`. Can you confirm?

I agree a test like this should not exist, but if it gets affected by this batch mode then something is not right.

Reply:
The test fails without any modifications: https://dev.azure.com/Lightning-AI/lightning/_build/results?buildId=80835&view=logs&j=3f274fac-2e11-54ca-487e-194c91f3ae9f&t=244491d3-5bd5-5b27-6d81-66bb4c7264ae&l=744

I only removed the `seed_everything` call after removing the convergence check